docs/2.7.0/docs/daml-finance/concepts/contingent-claims.rst

```diff
@@ -203,7 +203,19 @@ The Observation Parameter
 =========================
 
 ``o`` is the type used to represent ``Observation``\ s, which are time-dependent quantities that can
-be observed at any given time (such as the “EURUSD” exchange rate in the example above).
+be observed at any given time (such as the “EURUSD” exchange rate in the example above). There are
+two ways of creating an observation in the library:
+
+- :ref:`Observe <constr-contingentclaims-core-observation-observe-30391>` is the original Claims
```

Comment:
The paper doesn't really tell you anything about how to model an observation, so I wouldn't say it is the original operator :)